README.md

```diff
@@ -193,24 +221,52 @@ The following instructions assume a Unix-like shell (e.g. bash).
       go env GOROOT GOPATH
 
 - Ensure `$GOPATH/bin` is on your `$PATH`.
+
+- Clone the dcrdata repository. It **must** be cloned into the following directory.
+
+      git clone https://github.com/decred/dcrdata $GOPATH/src/github.com/decred/dcrdata
+
+Once Golang is installed and the dcrdata project is cloned proceed to the build steps.
+
+### Build from source for golang >= 1.11
+
+Golang 1.11 introduced a new [dependency management feature](https://github.com/golang/go/wiki/Modules) that
+negates the use of third party tooling such as `dep`. Using `Go mod` will be the dependency managment tool of choice
+for dcrdata and other Decred projects going forward.
+
+Usage is simple and nothing is required except Golang 1.11.
+
+1. `go build`
+
+If you want to pre staging dependencies without building you can run
```

Comment:
Reads weird.  Please adjust grammar.  Also, why and when would you do this (esp. in the context of dcrdata)?   `go build` does this automatically, so it's not clear why it's mentioned here.  IMO, `go mod vendor` might be worth mentioning to prepare a vendor folder for go 1.10 or a setup that is not using modules.